etcdserver/api/v3rpc/key.go

```diff
@@ -134,9 +134,15 @@ func checkPutRequest(r *pb.PutRequest) error {
 	if len(r.Key) == 0 {
 		return rpctypes.ErrGRPCEmptyKey
 	}
+	if r.IgnoreLease && r.IgnoreValue {
```

Comment:
i think this is OK? With both, only the revision is bumped. Anything reason for us to return error here?